apidef/oas/upstream.go

```diff
@@ -261,13 +266,41 @@ func (ppk PinnedPublicKeys) Fill(publicKeys map[string]string) {
 
 	i = 0
 	for _, domain := range domains {
-		ppk[i] = PinnedPublicKey{Domain: domain, List: strings.Split(strings.ReplaceAll(publicKeys[domain], " ", ""), ",")}
+		ppk[i] = PinnedPublicKey{Domain: domain, PublicKeys: strings.Split(strings.ReplaceAll(publicKeys[domain], " ", ""), ",")}
```

Comment:
Do we need to sanitize some inputs here? Are there patterns we shouldn't allow?

Reply:
ideally it should be public keys - do we need to add it here now?